crates/sdk/macros/src/logic/method.rs

```diff
@@ -215,6 +257,23 @@ impl<'a, 'b> TryFrom<LogicMethodImplInput<'a, 'b>> for LogicMethod<'a> {
             }
         }
 
+        let mut modifiers = vec![];
+
+        for attr in &input.item.attrs {
+            if attr.path().segments.len() == 2
+                && attr.path().segments[0].ident == "app"
+                && attr.path().segments[1].ident == "init"
+            {
+                if let Some(_self_type) = &self_type {
```

Comment:
another thing to check for is that the `init` method is `pub`lic

Reply:
https://github.com/calimero-network/core/blob/d107917304df78a1ee75c5d8ee7fd4702dc98cbd/crates/sdk/macros/src/logic/method.rs#L144

Is it fine because here I already make sure it is public?

Reply:
Aha, right.. we're not bothering to parse private methods